### PR TITLE
Updates Star Realms version to 2.31

### PR DESCRIPTION
**Pull request by deiga**

### CODE REVIEW

File: Casks/star-realms.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
   # cloudfront.net is the official download host per the vendor homepage
-  url "http://d3dyghjp0ukdwm.cloudfront.net/StarRealms_v#{version.gsub('.','_')}.dmg"
+  url "http://downloads.starrealms.com/StarRealms-v#{version.gsub('.','')}.dmg"
   name 'Star Realms'
   homepage 'http://www.starrealms.com/digital-game/'
   license :freemium
 
-  app 'Star Realms.app'
+  app 'StarRealms.app', :target => 'Star Realms.app'
```
Comment (vitorgalvao) [inline Casks/star-realms.rb]: Please [do not `:target` for no reason](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#target-should-only-be-used-in-select-cases).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ecafa0f84dd8b2f479a3474af9eb986321c478ab. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by deiga** [comment]
@vitorgalvao Now the Formula won't overwrite the old executable and the executable is named wrong :(

**Comment by vitorgalvao** [comment]
The executable is named right (i.e. as the developer intended). Yes, renames do that. We have better support for renames planned, but in the meantime we can’t sacrifice consistency.

**Pull request closed without merging**